From 89377f99947391c081df6dad27edf6ac3daec5c0 Mon Sep 17 00:00:00 2001
From: Jim Meyering <meyering@redhat.com>
Date: Tue, 28 Aug 2012 19:47:54 +0200
Subject: [PATCH 07/89] maint: avoid new syntax-check failure re @xref

* doc/parted.texi: Adjust @xref usage: it must be used only
at start of sentence.
* doc/parted-pt_BR.texi: Likewise.
---
 doc/parted-pt_BR.texi | 4 ++--
 doc/parted.texi       | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/doc/parted-pt_BR.texi b/doc/parted-pt_BR.texi
index 8eb5cec..8878f83 100644
--- a/doc/parted-pt_BR.texi
+++ b/doc/parted-pt_BR.texi
@@ -1094,7 +1094,7 @@ de sistemas operacionais Microsoft.
 
 @item raid
 (MSDOS) - este sinalizador pode ser habilitado para dizer ao Linux que
-essa partição é uma partição de software RAID @xref{LVM e RAID}.
+essa partição é uma partição de software RAID. @xref{LVM e RAID}.
 
 @item LVM
 (MSDOS) - este sinalizador pode ser habilitado para dizer à partição
@@ -1994,7 +1994,7 @@ mudan
 com o kernel 2.4, e quando nós adicionarmos suporte a ele.)
 
 Se você quer redimensionar sua partição root ou de boot, use um disco
-de boot @xref{Discos de boot do Parted}, ou use o redimensionador
+de boot @pxref{Discos de boot do Parted}, ou use o redimensionador
 online do Andreas Dilger, incluído no pacote ext2resize @ref{Ext2}.
 
 @menu
diff --git a/doc/parted.texi b/doc/parted.texi
index 6561d0e..1601151 100644
--- a/doc/parted.texi
+++ b/doc/parted.texi
@@ -348,7 +348,7 @@ specify the location in ``G''). Use the sector unit ``s'' to specify exact
 locations.  With parted-2.4 and newer,
 IEC binary units like ``MiB'', ``GiB'', ``TiB'', etc., specify
 exact locations as well.
-See @xref{IEC binary units}.
+@xref{IEC binary units}.
 
 If you don't give a parameter to a command, Parted will prompt you for it.
 For example:
-- 
1.8.5.3

